syntax = "proto3";

message AutoSearchReq {
  string dtu_sn = 1;
  int32 time = 2;
  int32 package_nub = 3;
  int32 package_now = 4;
  repeated int64 mi_sn = 5;
}

message AutoSearchRes {
  string ymd_hms = 1;
  int32 offset = 2;
  int32 package_now = 3;
  int32 err_code = 4;
  int32 time = 5;
}
